src/shared/utils/ReactNodeTypes.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactNodeTypes
+ */
+
+'use strict';
+
+var ReactElement = require('ReactElement');
+
+var invariant = require('invariant');
+
+var ReactNodeTypes = {
+  NATIVE: 0,
+  COMPOSITE: 1,
+  EMPTY: 2,
+
+  getType: function(node) {
+    if (node === null || node === false) {
```

Comment:
Perhaps this logic should be shared for guaranteed consistency? (instantiateReactComponent has it too)